### DISCUSSION TITLE
你们遇到虚拟table 左右滚动的时候老是触发浏览器的上一步，从而导致不能滚动的问题吗？

### DISCUSSION THREAD

**Discussion by congshu-du** [ant-design/ant-design#44962] [Q&A]
我的是mac, 同事用windows就没触发过，我触发的还挺频繁的。

**Comment by zombieJ** [comment]
滚动到边界的时候 MAC 会当做前后导航，幅度小就不会了。不过如果有更好代码的解法，欢迎交流~

**Reply by congshu-du** [reply]
不是滚动到边界，就是划不动，然后直接触发的后退。

**Reply by congshu-du** [reply]
直接用鼠标拖动滚动条就可以拖回来，然后再右滑没事，左滑就又划不动，触发上一页。

**Comment by kiner-tang** [comment]
可以尝试用这个方法禁止一下 chrome 左滑返回上一页的默认行为：https://zhuanlan.zhihu.com/p/481798545?utm_id=0
<img width="989" alt="image" src="https://github.com/ant-design/ant-design/assets/10286961/ac5db4ed-87c3-495c-8d8c-76d500c07cab">

**Comment by linxianxi** [comment]
为啥我试了官网的 demo 不会

**Reply by congshu-du** [reply]
官网好像把chrome上一页下一页都禁了。两指右滑都没办法后退。 所以触发不了

**Reply by linxianxi** [reply]
没有的，你鼠标放在官网 demo 外面，是可以触发上一页的。反正我官网的复现不了

**Reply by congshu-du** [reply]
还没一个条件，每次chrome有更新就容易触发，你升级chrome之后就好了，过两天chrome 又有更新就又会触发。

**Reply by congshu-du** [reply]
兄弟，我在官网复现了，录屏了，怎么发给你

**Reply by congshu-du** [reply]
https://codesandbox.io/s/xu-ni-lie-biao-antd-5-9-0-forked-wdg63t?file=/demo.tsx

**Reply by linxianxi** [reply]
我的不会，难道是浏览器版本或系统版本不一样？

**Reply by congshu-du** [reply]
也不是一直能复现，我这边要是找到必现的方法我再发给你

**Comment by congshu-du** [comment]
https://codesandbox.io/s/xu-ni-lie-biao-antd-5-9-0-forked-wdg63t?file=/demo.tsx

**Comment by Chasen-Zhang** [comment]
same +1

**Comment by li-jia-nan** [comment]
这样就好了
```css
html, body {
  overscroll-behavior: contain;
}
```
